src/QirRuntime/test/QIR-static/compiler/QirTarget.qs

```diff
@@ -5,6 +5,26 @@ namespace Microsoft.Quantum.Intrinsic {
 
     open Microsoft.Quantum.Targeting;
 
+    function NAN() : Double {
```

Comment:
Should these be marked `@Inline`? (Not a big deal, probably, because eventually we want to use compiler's target-specific qs files anyway.)

Reply:
Fixed.